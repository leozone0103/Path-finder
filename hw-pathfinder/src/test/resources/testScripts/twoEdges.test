# Test findPath between two nodes with multiple edges
CreateGraph graph

AddNode graph A
AddNode graph B

AddEdge graph A B 1.000
AddEdge graph A B 2.000
AddEdge graph B A 3.000
AddEdge graph B A 4.000

FindPath graph A B
FindPath graph B A