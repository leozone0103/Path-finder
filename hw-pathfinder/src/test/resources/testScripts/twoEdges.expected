# Test findPath between two nodes with multiple edges
created graph graph

added node A to graph
added node B to graph

added edge 1.000 from A to B in graph
added edge 2.000 from A to B in graph
added edge 3.000 from B to A in graph
added edge 4.000 from B to A in graph

path from A to B:
A to B with weight 1.000
total cost: 1.000
path from B to A:
B to A with weight 3.000
total cost: 3.000
